### ISSUE TITLE
Declare interfaces between frontend and backend

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#92]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* interfaces for following artifacts are declared/specified:
* DI/connection
* schema
* credential

* interfaces to create, get existing, edit and delete the artifact, optionally more

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by veitmo** [comment]
@annikakrause 
Hey Annika, habe hier mal ein Issue für die Schnittstellen-Deklaration angelegt. Falls dir weitere Schnittstellen einfallen, würde ich mich freuen, wenn du die ergänzen könntest.
